packages/babel-core/test/api.js

```diff
@@ -172,6 +174,38 @@ describe("api", function () {
     expect(options).toEqual({ babelrc: false });
   });
 
+  describe("browserslistConfigFile", function () {
```

Comment:
I really dislike this test, especially because you had to modify the code in `src` in order to make it work (exposing the `resetBlockHoistPlugin` function).

I think that this PR is an example of "no tests is ok", since it's just an option we forgot to pass.

Reply:
I removed the test.